docs/source/use_dataset.mdx

```diff
@@ -64,7 +64,12 @@ These values are actually the model inputs.
 >>> dataset = dataset.map(tokenization, batched=True)
 ```
 
-**4**. Set the format of your dataset to be compatible with your machine learning framework:
+***4**. Add the `labels` key with the current label info to match the columns of the current model:
+```py
+>>> dataset = dataset.map(lambda example: {"labels": example["label"]}, batched=True)
+```
+
```

Comment:
We can avoid introducing a new step by replacing `labels` mentions with `label` in the existing ones.